stable/oauth2-proxy/templates/deployment.yaml

```diff
@@ -66,6 +66,10 @@ spec:
           - --htpasswd-file=/etc/oauth2_proxy/htpasswd/users.txt
         {{- end }}
         env:
+        {{- range $key, $value := .Values.extraEnv }}
```

Comment:
if this could be a toYaml block it is more versatile, e.g. can refer values in configmaps/secrets

Reply:
Makes sense, thanks!